CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.7.0 (2019-11-15)
+
+- Added `tag` input to the repeater item component and `orderChange` output to the repeater component to allow tracking user updates to the repeater item order. [#91](https://github.com/blackbaud/skyux-lists/pull/91)
```

Comment:
Does this mean the same thing?

"Added the `tag` input and the `orderChange` output to the repeater component to track user updates to the repeater item order."

Reply:
Unfortunately, no. `tag` is on the item component, while `orderChange` is on the repeater component.

Reply:
Ah, my bad! I overlooked the word item after `tag`, sorry.